src/test/java/com/bootexample4/RoostTest/deeplinkPostTest.java

```diff
@@ -0,0 +1,96 @@
+// Test generated by RoostGPT for test test-Koreai using AI Type Vertex AI and AI Model code-bison-32k
+
+
+// Test generated for /deeplink_post for http method type POST in rest-assured framework
+
+
+// RoostTestHash=e18295dfea
+
+
+package com.bootexample4.RoostTest;
+import io.restassured.RestAssured;
+import io.restassured.http.ContentType;
+import io.restassured.response.Response;
+import org.junit.jupiter.api.Test;
+import static io.restassured.RestAssured.given;
+import static org.junit.Assert.*;
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.List;
+import org.hamcrest.MatcherAssert;
+import static org.hamcrest.Matchers.*;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class deeplinkPostTest {
+  
+    @Test  
+    public void deeplink_post_Test() {  
+        RestAssured.baseURI = System.getenv("BASE_URL");  
+  
+        // Read CSV file  
+        try (BufferedReader reader = new BufferedReader(new FileReader("src/test/java/com/bootexample4/RoostTest/deeplinkPostTest.csv"))) {  
+            String headerLine = reader.readLine();  
+            String[] headers = headerLine.split(",");  
+  
+            String line;  
+            while ((line = reader.readLine()) != null) {  
+                String[] data = line.split(",");  
+  
+                // Create a map of header to data  
+                Map<String, String> map = new HashMap<>();  
+                for (int i = 0; i < headers.length; i++) {  
+                    map.put(headers[i], data[i]);  
+                }  
+                
+  
+                Response response = given()
+				.pathParam("koreUserId", map.get("koreUserId"))
+				.pathParam("botId", map.get("botId"))
+				.pathParam("accountId", map.get("accountId"))
+				.pathParam("tenantId", map.get("tenantId"))
+				.pathParam("Authorization", map.get("Authorization"))
+				.pathParam("environment", map.get("environment"))
+				.pathParam("channel", map.get("channel"))
+				.contentType(ContentType.JSON)
+				.body("{\n"+
+					"  \"usecase\": \"" + map.get("usecase") + "\n" +
+ 				"}")
+                .when()
+                .post("/deeplink")  
+                .then() 
+                .extract().response();    
+         
+                if (response.statusCode() == 200) {
+					System.out.println("Description: success");
```

Comment:
no assertions added but on postman we are getting
[
    {
        "usecase": "addbeneficiary",
        "link": "www.google.com",
        "linkType": "deeplink"
    }
]